dunwoody_disaster/animations/idle.py

```diff
@@ -0,0 +1,27 @@
+import pygame
+from dunwoody_disaster.animations.PygameAnimation import PygameAnimation
+import dunwoody_disaster as DD
+
+# from typing import override
```

Comment:
@EngineeringDude123 let's try to update your python version and see if this issue goes away.